scripts/gobuild.sh

```diff
@@ -14,6 +14,7 @@
 # limitations under the License.
 set -x
 set -e
+export PATH=/usr/lib/go-1.6/bin:$PATH
```

Comment:
Is this something we want to hardcode like this? Especially in a non distro version specific script, this doesn't seem ideal.